### PR TITLE
Bugfix trello 1708 target region takes partial screenshot on mobile

### PR DESCRIPTION
**Pull request by skhalymon**

### CODE REVIEW

File: tests/functional/eyes_selenium/mobile/test_mobile_web_region.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import pytest
+from appium import webdriver
+from selenium.webdriver.common.by import By
+
+from applitools.selenium import StitchMode, Target
+
+
+@pytest.fixture
+def webdriver_module():
```
Comment (ikseek) [inline tests/functional/eyes_selenium/mobile/test_mobile_web_region.py:9]: Is it used?

File: tests/functional/eyes_selenium/mobile/test_mobile_web_region.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+@pytest.mark.platform("iOS")
+@pytest.mark.capabilities(
+    **{
+        "platformName": "iOS",
+        "browserName": "Safari",
+        "deviceName": "iPhone 11",
+        "platformVersion": "13.4",
+    }
+)
+@pytest.mark.test_page_url("https://applitools.com")
```
Comment (ikseek) [inline tests/functional/eyes_selenium/mobile/test_mobile_web_region.py:35]: Need a simple test page that will outlive applitools.com redesigns

File: eyes_selenium/applitools/selenium/webelement.py [hunk 3]
```diff
@@ -356,3 +359,95 @@ def __init__(self, width, height, left, top, right, bottom):
...
+class MobileSafariElementAdapter(ElementAdapter):
+    """
+    Some browser, like MobileSafari returns relative position parameters for element.
+    This is class helper converts relative position to absolute.
+    """
+
+    def __init__(self, eyes_element):
+        # type: (EyesWebElement) -> None
+        super(MobileSafariElementAdapter, self).__init__(eyes_element)
+        self._position_provider = ScrollPositionProvider(
```
Comment (ikseek) [inline eyes_selenium/applitools/selenium/webelement.py:405]: Unsued

File: eyes_selenium/applitools/selenium/positioning.py [hunk 4]
```diff
@@ -211,3 +212,44 @@ def get_entire_size(self):
...
+        )
+
+
+def create_position_provider(driver, stitch_mode, scroll_root_element, target_element):
+    # type: (EyesWebDriver,StitchMode,EyesWebElement,Optional[EyesWebElement])->PositionProvider
+    logger.debug("initializing position provider. stitch_mode: {}".format(stitch_mode))
+    if stitch_mode == StitchMode.Scroll:
+        return ScrollPositionProvider(driver, scroll_root_element)
+    elif stitch_mode == StitchMode.CSS:
+        if driver.user_agent.browser == BrowserNames.MobileSafari:
```
Comment (skhalymon) [inline eyes_selenium/applitools/selenium/positioning.py:249]: Add OS Name

**Pull request merged**